api/app/statehandler.py

```diff
@@ -49,7 +49,7 @@ def is_command_result_topic(self, topic: str) -> bool:
 
     def recieved_announcement(self, message) -> DeviceDTO:
         device = DeviceDTO.parse_raw(message)
-        deviceId : str = device.identifier 
+        deviceId : str = device.device_id 
```

Comment:
there are more device.identifier references - eg hass formatter

Reply:
Good catch, searched the code base for ".identifier" without any hits so I think that was the last one.